content/en-US/api/2dcontextlostevent.md

```diff
@@ -0,0 +1,30 @@
+---
+recipe: api-interface
+title: '2dcontextlost'
+mdn_url: /en-US/docs/Web/API/2dcontextlost_event
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.2dcontextlost
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The `contextRestored` event is a javascript event that is triggered when the user agent detects
+that the context associated with 'CanvasRenderingContext2D' on the page is "lost". Contexts can
+be lost for several reasons, such as a driver crashes, the application runs out of memory, etc.
```

Comment:
At the risk of sounding like a pest, this has to follow the MDN wording *exactly*. The API section is a reference like a dictionary. Structural and lexical consistency support quick scanning for information. [Here's an example of what the wording should like.](https://developer.mozilla.org/en-US/docs/Web/API/MIDIPort/onstatechange)